.github/workflows/e2e_tests.yml

```diff
@@ -105,10 +105,110 @@ jobs:
           gsutil cp "${UPSTREAM_GCSP_PATH}/stats-${SHA}*" ${RELEASE_GCS_PATH}/
           gsutil cp "${UPSTREAM_GCSP_PATH}/attributegen-${SHA}*" ${RELEASE_GCS_PATH}/
 
-  create-test-images:
-    name: create-test-images
-    runs-on: ubuntu-latest
-    needs: [build_fips_proxy]
+  build_arm_proxy:
+    # Do not run this job when the release type equals "fips".
```

Comment:
add comment with reason? because we don't suppor fips arm combination yet?

Reply:
done